WTWSMS/history/provinces/676 - Kakheti.txt

```diff
@@ -15,8 +15,12 @@ b_davidgareja = temple
 #b_gremi = castle
 
 # Misc
-culture = armenian
-religion = georgian_orthodox
+culture = juhuro
+religion = tzadikite
```

Comment:
My reading of the sources is that there was an Armenian Georgian-Christian majority at this point of time.